libraries/cli/src/config.cpp

```diff
@@ -247,6 +252,26 @@ bool Config::nodeConfigured() { return node_configured_; }
 
 FullNodeConfig Config::getNodeConfiguration() { return node_config_; }
 
+void Config::addDebugLoggingConfiguration(Json::Value& logging_config) {
+  constexpr std::string_view debug_config_json = R"({
+      "name": "debug",
+      "on": false,
+      "verbosity": "DEBUG",
+      "outputs": [
+        {
+          "type": "file",
+          "file_name": "debug/TaraxaDebug_N1_%m%d%Y_%H%M%S_%5N.log",
+          "rotation_size": 10000000,
+          "time_based_rotation": "0,0,0",
+          "format": "%ThreadID% %ShortNodeId% %Channel% [%TimeStamp%] %SeverityStr%: %Message%",
+          "max_size": 1000000000
+        }
+      ]
+    })";
```

Comment:
how does this work for other configurations e.g. --log-configurations "trace"? or specific channel  as per help msg `Log confifugrations to use: [channel:level, ....]`

Reply:
I didn't notice that, but as I see it is just wrong help message. So It just enables predefined configuration with name you are specifying (see [`tools.cpp:106`](https://github.com/Taraxa-project/taraxa-node/blob/1cbb8bd9ff5b30f92a0bf588864c6b1868281eb6/libraries/cli/src/tools.cpp#L106)).